mreg/models.py

```diff
@@ -451,6 +451,22 @@ def zf_string(self, zone):
         return '{name:24} {ttl:5} IN {record_type:6} {priority} {weight} {port} {target}\n'.format_map(data)
 
 
+class HostGroup(models.Model):
+    hostgroup_name = models.CharField(max_length=50, unique=True)
+    parent = models.ManyToManyField('self', symmetrical=False, blank=True, related_name='groups')
+
+    def __str__(self):
+        return("%s" % (self.id))
+
+
+class HostGroupMember(models.Model):
+    hostid = models.ForeignKey(Host, on_delete=models.PROTECT, db_column='hostid', related_name='hostgroupmember')
```

Comment:
related_name should be plural. Also we generally use `host = models.Foreignkey(Host, ...`